apps/api/src/domain/models/song/queries/song.view-model.event.sourced.ts

```diff
@@ -0,0 +1,115 @@
+import { FromDomainModel, NonNegativeFiniteNumber, URL } from '@coscrad/data-types';
+import { isNonEmptyObject, isNonEmptyString } from '@coscrad/validation-constraints';
+import { CoscradDataExample } from '../../../../test-data/utilities';
+import { DTO } from '../../../../types/DTO';
+import { buildMultilingualTextWithSingleItem } from '../../../common/build-multilingual-text-with-single-item';
+import { MultilingualText } from '../../../common/entities/multilingual-text';
+import buildDummyUuid from '../../__tests__/utilities/buildDummyUuid';
+import { AccessControlList } from '../../shared/access-control/access-control-list.entity';
+import { ContributionSummary } from '../../user-management';
+import { SongCreated } from '../commands';
+import { Song } from '../song.entity';
+
+const FromSong = FromDomainModel(Song);
```

Comment:
I'm wondering if we want to keep doing this. It doesn't really save that much and it scares me that we could have circular deps.